### DISCUSSION TITLE
Ender 3 S1 CR Touch automatic bed leveling problem

### DISCUSSION THREAD

**Discussion by coderm4ster** [mriscoc/Ender3V2S1#1016] [Q&A]
I don't know if anyone can help me with what could be wrong with the automatic bed leveling. I tried with the professional firmware versions 20221002 and 20230522, but both have the same result.

So, I tried to level the front two corners of the bed (and only those two) using the automatic bed tramming function. First, I unchecked the Manual Tramming option. Then I adjusted the bed leveling screws first at the front left corner and then at the front right corner, while after each adjustment I measured the Z distance using the Front Left and Front Right menu items.
After measuring the same Z distance at both corners (in my case -0.15) based on CR Touch, I switched to manual bed leveling, traditionally using paper. To do this, I checked the Manual Tramming option, then tried to check the two corners again using the two menu items mentioned above (in this case, the nozzle is in the same position where the CR Touch sensor was during the automatic leveling). The result was that, while at the left corner the paper barely touched the nozzle, at the right corner the nozzle pressed the paper against the bed.

I don't understand why, while according to CR Touch the nozzle is at the same distance from the bed in both corners, according to the paper, the nozzle in the left corner is several tenths of a millimeter further from the bed than in the right corner. What is the difference between the two methods? In my opinion, the paper should pull equally in both corners.

I checked the CR Touch attachment, the tightness of the head and the bed, they are all good, no wobbles. I also turned off the HS mode to see if it affected it, but it didn't.
I use a 4 mm glass sheet on the bed, so I wouldn't even say that some kind of unevenness bothers me.

Does anyone have any idea what could be causing this problem?

Unfortunately, this completely prevents me from using automatic bed leveling (not to mention automatic mesh generation).

Thanks if anyone can give me an idea.

**Comment by mriscoc** [comment]
Check https://github.com/mriscoc/Ender3V2S1/issues/614

**Reply by coderm4ster** [reply]
Thank you, I read through the linked issue and it makes me wonder why this probe mod works. Anyway I will try it, although my printer has F1 firmware but I hope it will work.

I will report back after I have tried it.

**Reply by coderm4ster** [reply]
I printed out one of the probe relocation mods, installed it and with different settings (HS mode on/off, Z feed rate 480-960) I made 5x5 and 9x9 meshes and printed a full bed covering first layer square.
While printing it seemed to use the mesh, the Z offset icon flashed. Unfortunately the results were no better, the full square had problems, the perfect first layer failed.

I'm disappointed, this printer is a lost axe handle...

![20230814_163115](https://github.com/mriscoc/Ender3V2S1/assets/14206860/75c138fb-9ffe-4760-a290-193fbc733ccb)
